docs/integrations/containers-orchestration/kubernetes.md

```diff
@@ -322,10 +323,10 @@ Use this dashboard to:
 
 ### Health Check
 
-The **Kubernetes - Health Check**dashboard displays the collection status from all the components in the Kubernetes cluster.
+The **Kubernetes - Health Check** dashboard displays the collection status from all the components in the Kubernetes cluster.
 
 Use this dashboard to:  
-* Monitor the health of FluentD and FluentBit pods in your Kubernetes environment.
+* Monitor the health of FluentD and FluentBit pods in your Kubernetes environment (note: currently in default configurations [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) is used instead of FluentD and Fluent Bit)
```

Comment:
Later we should change this dashboard to present otelcol intead of Fluentd and Fluent Bit, jira ticket for this:  https://jira.kumoroku.com/jira/browse/SUMO-214067